### PR TITLE
feat(rating): add built-in translations

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #4961

## Summary

Wires up translations to the rating component.

### CODE REVIEW

File: src/tests/commonTests.ts [hunk 1]
```diff
@@ -974,25 +974,35 @@ export async function t9n(componentSetup: TagOrHTML | TagAndPage): Promise<void>
...
+      fakeBundleIdentifier
+    );
 
     component.setAttribute("lang", "es");
     await page.waitForChanges();
     await page.waitForTimeout(3000);
     const esMessages = await getCurrentMessages();
 
-    expect(esMessages).toEqual({});
+    expect(esMessages).toHaveProperty(fakeBundleIdentifier);
```
Comment (anveshmekala) [inline src/tests/commonTests.ts:1005]: we can assert `esMessages` toEqual `enMessage` without the need of adding `fakeBundleIdentifier`.

**Review by anveshmekala — APPROVED** [review]
LGTM! 💯

File: src/tests/commonTests.ts (hunk 1 above)
Comment (jcfranco) [reply src/tests/commonTests.ts:1005]: If we did so, this would no longer catch problems with locale bundle switching as the `en` one wouldn't change.

Comment (anveshmekala) [reply src/tests/commonTests.ts:1005]: make sense.

**Pull request merged**